src/test/java/br/unb/cic/analysis/samples/ioa/BaseNotConflictSample.java

```diff
@@ -0,0 +1,28 @@
+package br.unb.cic.analysis.samples.ioa;
+
+// Not Conflict
+public class BaseNotConflictSample {
+
+    public String text;
+    public int fixes;
+
+    public int countFixes() {
+        BaseNotConflictSample baseConflictSample = new BaseNotConflictSample();
```

Comment:
@barbosamaatheus teria como evitar esse new, e as chamadas serem com this ou mesmo omitindo this, para ficar mais realista e igual ao exemplo do slide?

Reply:
Não, pelo que me lembro quando começamos a usar o call graph ele precisa de uma instância da classe, por isso fazemos isso. É por causa disso que precisamos fazer algumas mudanças realistas nos cenários. Mas vou investigar melhor isso e ver se temos outra solução mais elegante.

Reply:
talvez criando um main na classe que faz isso

Reply:
@pauloborba essa foi a melhor forma que encontrei. Estou usando o método com o nome de callRealisticRun, pois é o mesmo método que uso nas execuções realistas. Desta forma a análise entende que precisa usar esse método como entry point e cria a instância da classe.